src/app/shared/issue/label/label.component.html

```diff
@@ -15,11 +15,15 @@
       <mat-icon [ngStyle]="{ color: '#' + value.labelColor, 'font-size': '1.8em' }">stop</mat-icon>
       <span> {{ value.labelValue }}</span>
     </button>
-
     <button *ngIf="hasLabelDefinition(value)" class="infoPopup" (click)="openDefinitionPage(value); $event.stopPropagation()">
       <mat-icon style="font-size: 20px">info</mat-icon>
     </button>
   </div>
 </mat-menu>
 
-<p [ngStyle]="this.labelService.setLabelStyle(this.labelColor)">{{ issue[attributeName] || '-' }}</p>
+<p
+  [ngStyle]="this.labelService.setLabelStyle(this.labelColor)"
+  [matTooltip]="this.labelService.parseLabelDefinitionMarkdown(this.labelService.getLabelDefinition(issue[attributeName], attributeName))"
+>
+  {{ issue[attributeName] || '-' }}
+</p>
```

Comment:
Is it necessary to include tooltips for labels in the label selection? Label definitions can already be viewed when selecting them.

Reply:
Currently, the tooltips only apply to the colored labels, and not within the selection menu as shown below:

![image](https://github.com/user-attachments/assets/ec30635e-3e2e-41bf-bc1e-d94f1ed7e416)
![image](https://github.com/user-attachments/assets/c7023415-41e7-40f3-9272-af83459b48b7)

As for why I support having the tooltips in these components, it is because when I was using CATcher during my run of CS2103T, I remember there were some label definitions which were quite long and hard to remember. For example:

![image](https://github.com/user-attachments/assets/4fc51135-79cc-4447-b7f6-eb445a4994cf)

While the user may have already read the tooltips before selecting the option, the tooltips provide a more convenient way for them to view the definition after it has been selected. This will be especially useful when users are, for example, verifying the accuracy of the label which they have chosen to their bug report (which is quite important to them during their PE)